docs/orchestrating-elastic-stack-applications/elasticsearch/bundles-plugins.asciidoc

```diff
@@ -25,11 +25,37 @@ This option has the following pros and cons:
 This option has the following pros and cons:
 +
 * Pro: Easier to get started and upgrade versions.
-* Con: Requires pods to have internet access.
+* Con: Requires pods to have internet access. *See <<istio-note,note>> below about using Istio*
 * Con: Adding new Elasticsearch nodes could randomly fail due to network issues or bad configuration.
 * Con: Each Elasticsearch node needs to repeat the download, wasting bandwidth and slowing startup.
 * Con: Deployment manifests are more complicated.
 
+[id="istio-note"]
+**Note when using Istio**
+
+When using Istio, init containers do *not* have network access, as the Envoy sidecar that provides network connectivity is not online.  In this scenario, custom containers are the best option.  If custom containers are simply not a viable option, then it is possible, although not recommended to adjust the startup command for the elasticsearch container itself to run the plugin installation before starting Elasticsearch, as the following example describes.
```

Comment:
```suggestion
When using Istio, init containers do *not* have network access, as the Envoy sidecar that provides network connectivity is not started yet.  In this scenario, custom containers are the best option. If custom containers are not a viable option, then it is possible to adjust the startup command for the elasticsearch container itself to run the plugin installation before starting Elasticsearch, as the following example describes. Note that this approach will require updating the startup command if it changes in the Elasticsearch image.
```

Assuming that user can't use custom containers I think this would be our recommended option - the wording right now sounds very discouraging though :) I'd openly mention the downside of this approach instead.

Reply:
I adjusted this wording, along with adding a note about when this could potentially affect a user.